Program.cs

```diff
@@ -647,7 +653,7 @@ public void itemShop()
                 cottonList.Add(new Text("Cotton ", Colours.Cotton, TypeWriter.Speed.Talk));
                 cottonList.Add(new Text("and this is my shop, whats your name?", Colours.Speech, TypeWriter.Speed.Talk));
                 TypeWriter.WriteLine(cottonList);
-                cottonUserName = Console.ReadLine();
+                cottonUserName = GetAwnserThenTurnLower();
```

Comment:
Do you want the user name to be converted to lower case? It is really only the choices that we are looking to make sure will work for upper and lower case.